README.md

```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/teamdigitale/italia-ts-commons.svg?style=svg)](https://circleci.com/gh/teamdigitale/italia-ts-commons)
+[![CircleCI](https://circleci.com/gh/pagopa/io-ts-commons.svg?style=svg)](https://circleci.com/gh/pagopa/io-ts-commons)
```

Comment:
Maybe we have to remove all reference to CircleCi, since we are replacing it with Azure Pipelines :)

Reply:
Thanks, I was wondering what was the purpose of that (and it was blocking me!)